monkey/monkey_island/cc/server_setup.py

```diff
@@ -45,32 +44,35 @@
 
 def run_monkey_island():
     island_args = parse_cli_args()
-    config_options, server_config_path = _setup_data_dir(island_args)
+    config_options = _extract_config(island_args)
+    _setup_data_dir(config_options.data_dir)
 
     _exit_on_invalid_config_options(config_options)
 
     _configure_logging(config_options)
-    _initialize_globals(config_options, server_config_path)
+    _initialize_globals(config_options.data_dir)
 
     mongo_db_process = None
     if config_options.start_mongodb:
         mongo_db_process = _start_mongodb(config_options.data_dir)
 
     _connect_to_mongodb(mongo_db_process)
 
-    _configure_gevent_exception_handling(Path(config_options.data_dir))
+    _configure_gevent_exception_handling(config_options.data_dir)
     _start_island_server(island_args.setup_only, config_options)
 
 
-def _setup_data_dir(island_args: IslandCmdArgs) -> Tuple[IslandConfigOptions, str]:
+def _extract_config(island_args: IslandCmdArgs) -> IslandConfigOptions:
     try:
-        return config_setup.setup_server_config(island_args)
-    except OSError as ex:
-        print(f"Error opening server config file: {ex}")
-        exit(1)
+        return get_server_config(island_args)
     except json.JSONDecodeError as ex:
         print(f"Error loading server config: {ex}")
         exit(1)
```

Comment:
Since we're not getting import errors on any of the deployments I'd say we can use `exit`. I also don't mind switching to `sys.exit()`, but then we need to add this to our guidelines

Reply:
In terms of the changes, this already is `sys.exit()`, look at the import statements: `from sys import exit`

Reply:
Maybe just `import sys` and call `sys.exit()` then to avoid confusion and surprises.